pkg/ecdsa/dkg/gen/pb/message.proto

```diff
@@ -0,0 +1,15 @@
+syntax = "proto3";
+
+option go_package = "pb";
+package dkg;
+
+message EphemeralPublicKeyMessage {
+    uint32 senderID = 1;
+    map<uint32, bytes> ephemeralPublicKeys = 3;
```

Comment:
should the id here be 2? I know it doesn't matter in terms of the specification, but for readability/confusion

Reply:
Definitely! Looks like I made a typo. See 1ca5fe07